packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -5149,11 +5149,16 @@ curl -v -X POST \
 #### Verify WebAuthn Factor
 
 
-<ApiOperation method="post" url="/api/v1/authn/factors/${factorId}/verify" /> <SupportsCors />
+<ApiOperation method="post" url="/api/v1/authn/factors/${factorIdOrFactorType}/verify" /> <SupportsCors />
 
 Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options, which are used to help select an appropriate authenticator using the WebAuthn API.
 This authenticator then generates an assertion that may be used to verify the user.
 
+> **Note:** a `factorId` or `factorType` may be specified for WebAuthn's verify endpoint, as the WebAuthn factor type supports multiple factor instances.
+  When a `factorId` is used, the verification procedure is no different from any other factors, with verification for a specific factor instance.
+  When "webauthn" (the `factorType` name for WebAuthn) is used, verification would be acceptable with any WebAuthn factor instance enrolled for the user.
+
+
```

Comment:
nit: should this be 2 lines?  Not sure on docs

Reply:
Yeah, it's kind of arbitrary with the spacing in docs (there are a lot of random double-newlines on small sections and single-newlines on big sections), but it's kind of arbitrary since markdown only really registers the initial newline, I think? I can change it, though.

Reply:
Meh, it's fine to me as long as it shows up right